pennylane_lightning/src/util/Kokkos_Sparse.hpp

```diff
@@ -79,8 +79,7 @@ using const_data_view_type =
  * @brief Create a Kokkos Sparse Matrix object with unmanaged views.
```

Comment:
Could you please check the followings,
-  What does this "The j element encodes the number of non-zeros above row j" refer to at lines 82, 115, 168, 201?
- line 85 was that auto-formatted?

Reply:
In order:

- Let me know if it is clear now.
- Yes. It was.